test/e2e/runner/rpc.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	WaitForHeightTimeout = 20 * time.Second
+	WaitForHeightTimeout = 30 * time.Second
```

Comment:
is this still needed or added for debugging?

Reply:
20 sec was not enough for passing tests